### PR TITLE
prepare for sglang+verl

### PR DESCRIPTION
**Pull request by lbk-sys**
## Motivation
run sglang for verl

At that time, the NPU commercial law version was unable to support the adaptation of the current UUID scheme, so the change was temporarily avoided. It is expected that a replacement scheme will be available by the end of October.

## Modifications

At present, this PR is used to integrate verl+sglang+npu capabilities, while other modifications are incorporated into sgl kernel npu.

## Checklist

- [ ] Format your code according to the [Format code with pre-commit](https://docs.sglang.ai/developer_guide/contribution_guide.html#format-code-with-pre-commit).
- [ ] Add unit tests according to the [Run and add unit tests](https://docs.sglang.ai/developer_guide/contribution_guide.html#run-and-add-unit-tests).
- [ ] Update documentation according to [Write documentations](https://docs.sglang.ai/developer_guide/contribution_guide.html#write-documentations).
- [ ] Provide accuracy and speed benchmark results according to [Test the accuracy](https://docs.sglang.ai/developer_guide/contribution_guide.html#test-the-accuracy) and [Benchmark the speed](https://docs.sglang.ai/developer_guide/contribution_guide.html#benchmark-the-speed).

### CODE REVIEW

File: python/sglang/srt/torch_memory_saver_adapter.py [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
 from contextlib import contextmanager, nullcontext
+from sglang.srt.utils import (
+    is_npu,
+)
 
 try:
     import torch_memory_saver
 
     _memory_saver = torch_memory_saver.torch_memory_saver
+    if is_npu():
```
Comment (ping1jing2) [inline python/sglang/srt/torch_memory_saver_adapter.py]: `_is_npu = is_npu()`
